app/models/manageiq/providers/microsoft/infra_manager/provision/cloning.rb

```diff
@@ -156,7 +156,7 @@ def start_clone(_clone_options)
     if results.stdout.blank?
       raise MiqException::MiqProvisionError, results.stderr
     else
-      $scvmm_log.debug(update_vm_script)
+      $scvmm_log.debug(update_vm_script(results.stdout))
```

Comment:
Can we just call `script  = update_vm_script(results.stdout)` once then log `$scvmm_log.debug(script)` ?

Reply:
Done!